src/javascript/app/Services/ws-methods.js

```diff
@@ -146,4 +147,4 @@ const WS = (() => {
     };
 })();
 
-export default WS;
+export default trackJSNetworkMonitor(WS);
```

Comment:
is this supposed to be reverted to WS?

Reply:
Uh no. This is a proxy to catch `response.error`. It doesn't/shouldn't change the behavior of the `WS` module. it just exposes them to trackJS.